code/modules/mob/living/carbon/alien/humanoid/caste/sentinel.dm

```diff
@@ -45,8 +45,10 @@
 	if(name == "alien sentinel")
 		name = text("alien sentinel ([rand(1, 1000)])")
 	real_name = name
-	verbs.Add(/mob/living/carbon/alien/humanoid/proc/corrosive_acid,/mob/living/carbon/alien/humanoid/proc/neurotoxin)
-	..()
+	internal_organs += new /obj/item/organ/internal/xenos/plasmavessel

```

Comment:
yeah in its parent New() in alien.dm it adds organs via 
for(var/obj/item/organ/internal/I in internal_organs)
 I.insert(src)